README.md

```diff
@@ -140,6 +140,54 @@ env:
   SET_PUBLISH_DATE: true
 ```
 
+## Multiple shows per repository
+
+It is possible to use a single repository to maintain several shows.
+
+You'll need an episode config per show.
+
+As an example, suppose you have two shows, you called "Great News" and another "Sad News".
+
+You repository will look like this:
+
+```
+.github/
+├─ workflows/
+│  ├─ great-news.yaml
+│  ├─ sad-news.yaml
+great-news.json
+sad-news.json
+```
+
+In `great-news.json` and `sad-news.json`, you have:
+```json
+{
+  "id": "episode_video_id"
+}
+
+In `great-news.yaml`:
```

Comment:
Could you do something with the formatting of the block of code here?

It doesn't look right, the background of the text is red and is not readable.

Reply:
done.